python-package/xgboost/spark/model.py

```diff
@@ -222,11 +216,11 @@ def saveImpl(self, path):
         """
         xgb_model = self.instance._xgb_sklearn_model
         _SparkXGBSharedReadWrite.saveMetadata(self.instance, path, self.sc, self.logger)
-        model_save_path = os.path.join(path, "model.json")
-        ser_xgb_model = serialize_xgb_model(xgb_model)
-        _get_spark_session().createDataFrame(
-            [(ser_xgb_model,)], ["xgb_sklearn_model"]
-        ).write.parquet(model_save_path)
+        model_save_path = os.path.join(path, "model")
+        xgb_model_file = dump_model_to_json_file(xgb_model)
+        # The json file written by Spark base on `booster.save_raw("json").decode("utf-8")`
```

Comment:
Why not?

Reply:
There are some " \" " in the json file which can't be loaded by xgboost. Do you want to check more?

Reply:
I will take a look tomorrow

Reply:
@trivialfis No need anymore, I just found another way to do it.